docs/source/index.rst

```diff
@@ -20,6 +20,7 @@ Contents
     figures
     models
     simulation
+    subplot
```

Comment:
I think it's better to have one url for all figure/plot classes. What do you think?
So I would suggest to remove the subplot file and add the content to figures.